pennylane_sf/tf.py

```diff
@@ -273,3 +272,81 @@ def probability(self, wires=None):
         ind = np.indices([cutoff] * N).reshape(N, -1).T
         probs = OrderedDict((tuple(k), v) for k, v in zip(ind, probs))
         return probs
+
+    def jacobian(
+        self, queue, observables, parameters
+    ):  # pylint: disable=missing-function-docstring
+        op_params = {}
+        new_queue = []
+        variables = []
+
+        with tf.GradientTape(persistent=True) as tape:
+            for operation in queue:
+                # Copy the operation parameters to the op_params dictionary.
+                # Note that these are the unwrapped parameters, so PennyLane
+                # free parameters will be represented as Variable instances.
+                op_params[operation] = operation.data[:]
+
+            # Loop through the free parameter reference dictionary
+            for _, par_dep_list in parameters.items():
+                if not par_dep_list:
+                    # parameter is not used within circuit
+                    v = tf.Variable(0, dtype=tf.float64)
+                    variables.append(v)
+                    continue

```

Comment:
Nice! I haven't really used `continue` much, but it seems quite handy.

Reply:
Yeah! Typically any pattern of the form

```python
for i in sequence:
    if condition:
        do_thing()
    else:
        do_other_thing()
```

can be re-written as

```python
for i in sequence:
    if condition:
        do_thing()
        continue
    
    do_other_thing()
```

Comes down to personal preference :) If the `else` is particular large, I tend to favour the latter just to avoid the additional indentation.